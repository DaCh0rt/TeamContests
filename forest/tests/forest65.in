847 548
5611 -8420
8158 -9110
-4733 4724
-2919 -1147
4178 -970
-7826 -7379
-6433 -2342
-2088 1810
902 -2010
5124 9883
-9990 -6449
1628 -1330
-5772 -8298
-321 -4297
9709 6704
-9418 9364
-9314 -7110
8441 -5307
4978 5385
-5004 -7082
-3937 3473
1558 2544
-8116 2703
-9644 -4950
-6891 7793
-6154 4085
-3964 2115
2615 -621
-2715 9518
-8049 -8409
12 -7424
2834 7153
6444 -590
2069 -9347
5017 -3306
6532 5951
1654 -405
491 -3882
1576 -1378
-155 -2867
5122 -3033
572 -7995
-55 7788
5425 -4613
-4403 -9775
-5252 9629
5504 -5392
-9563 -4922
8376 -3613
1403 912
9301 5964
5510 2873
6980 -7647
582 7882
-9461 -9780
987 1543
2630 2803
5894 -8863
622 -6671
-3378 2586
7383 -8802
-4610 7398
1154 7325
-8216 -4059
6454 3003
8626 844
-2777 -4057
5983 6511
4466 8789
-349 -4053
-6325 9096
-4639 5051
3405 -6126
-4976 -9658
5882 -833
8501 -6479
8747 -5851
7273 -8785
1109 -4500
7813 -4015
9486 9924
4118 452
3649 2623
-4943 4322
-74 1218
4708 8064
7940 9923
-4297 5072
-8232 4647
6872 3136
-4825 -1367
-3627 6414
9756 -3470
6706 7943
-8158 8160
8068 -8210
5126 -1496
-6030 -3287
868 -6010
-292 -388
3570 9184
-8099 6737
-2034 -3619
-4732 -9184
-1470 -1093
-6946 4889
6013 6783
-6498 1074
-6552 579
6465 844
-9172 1916
850 -6147
3884 9630
320 7260
-1196 3845
-9250 3257
-6812 3174
1169 8379
-6681 -60
-3324 -5318
639 1019
4224 -2883
-4456 -4100
-4618 -7604
-9387 1658
-7505 3937
-2146 8206
9815 -2652
2791 -1603
-1931 217
-4301 5630
3924 -423
2304 -4289
100 -2681
9833 -4502
3532 -1801
-3817 -4998
8761 -970
2820 -1600
7962 -8961
5905 3548
-6711 7723
-7973 4455
6007 -1861
2442 -2022
-9900 -2995
-911 436
-8888 2521
-741 653
-6535 -5310
-5204 -5826
7547 4574
3478 5695
6461 9746
9950 4070
9738 2420
-3557 790
5457 7060
-1405 9570
5155 4680
9312 -4762
6843 -2789
-7285 -2223
998 1071
5649 642
-8816 874
-935 207
-2042 9543
-6525 4162
-2916 -1747
7323 4827
654 61
-3138 -2650
-1367 2609
-5404 -3231
7616 1926
1646 9005
7813 4480
-488 4354
6886 -1993
9258 7196
2311 2395
-9960 384
8124 -6362
8833 -5564
7089 -8525
-5307 -4078
-6118 335
-7476 -8666
8434 -4142
-8109 -1136
5805 9411
-8460 3927
-3161 5530
-3956 3391
4463 1934
2118 -5856
-15 -9060
-268 -7615
5686 -4650
1057 1164
-9628 -7076
-5845 -7382
7354 -6818
6014 2781
6435 1827
3572 -8333
-1586 -2566
3875 8786
-5267 6802
9359 -2522
5454 -6127
-6821 -6910
4104 -6143
4661 -2765
5907 -8735
-3464 -7968
712 -6205
-9615 6806
-5333 3024
542 -8969
-6071 8399
-1945 9902
1532 8204
9322 -9798
9194 8443
-5463 2173
2711 2165
-6698 7862
-1831 5096
-7015 -2762
-7315 5887
-6713 2632
9200 -7887
-9508 -4532
-6655 2578
6808 -504
-7273 2313
629 -4813
-4475 167
9264 -7720
5579 -4875
-2918 -9440
-1019 -622
-3036 -7107
4125 3377
1358 5139
3798 -609
-4448 -8652
-3007 -3000
-759 8349
-7027 -845
892 9213
144 8965
4566 484
369 -5638
-65 7919
9073 1553
-5170 -3863
3715 -2343
5065 9985
1811 3370
-2197 3854
-3928 7228
1717 7975
6233 -5227
2181 -2080
-2457 757
-4993 1240
1525 3568
9526 6176
-5669 -6314
806 -6609
2394 7720
-8311 984
-6346 138
-704 3303
-2165 4668
8996 5131
-2163 5345
-6251 -3889
1796 341
-7057 -1359
-4352 -5644
-3324 2516
252 8484
-7393 -6454
-9418 -9480
7780 -9285
5938 -2050
-3460 -8775
7360 5306
-1384 6622
808 -1842
-7394 3439
4320 -3375
3002 1652
2696 -1273
-9801 9763
6965 6001
-1948 -8783
-1771 -2145
-1456 -2796
4020 -6221
8630 7951
-4190 -476
1147 4283
-7786 3358
1254 -6919
4250 -1824
5349 -9368
5800 -7495
-9976 -9039
8582 4954
9952 -265
-8277 -3398
77 6065
-7648 8166
-5020 1989
-9944 -5985
-4457 6790
-5439 -5600
-7793 -5272
3996 -8739
7235 2242
-9161 153
3269 6637
-3300 -1452
6647 -5436
-2629 4337
-921 -1636
1953 124
7870 3527
6865 7650
1289 5025
-6128 6396
1480 353
3980 4070
-9480 9259
-3986 -2262
-5418 7577
-8098 6318
-5737 -1836
-7060 5670
-4187 -5273
-8452 -9346
5018 -666
-5660 -7339
3253 5523
-4814 -3706
-5393 -56
-3693 2024
-6667 -4005
-7745 -7241
9734 -4616
4588 -9959
4029 7118
3115 -2210
-723 8440
1936 -3791
3353 1615
7633 8387
4919 -6610
473 -8597
-4628 -4382
2255 1538
4773 9538
-5091 1295
-6170 -7973
-4578 -9216
-2133 -4238
6302 6870
-9791 -3328
8732 4518
639 8843
-1556 3381
-8872 -2787
4796 -8938
-3961 2292
9854 -3587
3939 -8296
-1791 -7269
4085 2093
-5124 -2004
7996 -9633
-6579 -1884
-5900 561
-3301 -1940
-2941 5301
-1217 7538
1833 -886
9416 8692
-9771 9907
-528 -2886
-3045 -4412
-9030 -4932
5043 -2180
7817 5447
2967 3356
2154 3174
1553 -2716
3241 -908
-5357 -1675
-2795 5390
-6838 7609
6091 -3508
725 8202
3378 6164
-3833 -6232
-1830 1560
9873 7161
-3095 537
-8754 3772
-1589 -2090
7862 2754
-4209 -4877
-5272 3511
7100 1904
5928 -5363
9329 -1240
-6114 -6921
-6575 7637
5421 3306
-7947 8094
7493 -5753
4551 1503
2847 -5898
-1462 7619
-7182 3812
-1593 629
-5398 -5820
2806 -3873
-7403 3916
-1993 -444
-4530 -4409
7543 7983
1078 -200
-4109 -3043
2584 5898
9880 597
3249 -4822
-4177 -8672
-3350 9273
3469 -8910
9200 5941
4735 8562
-2601 -6477
-466 -5350
-7105 -3315
-5824 969
-4859 -6037
-7910 -7443
4665 -6032
1790 -8643
2336 -1863
1707 7064
4045 8373
-2903 5476
8222 -3851
1002 -8582
2937 7825
7972 8174
-178 8478
61 -3487
-877 -7397
4113 1733
4809 -6279
5216 4530
-4187 160
-7316 -6409
-2736 -7130
4290 -9470
5350 -1487
-2465 504
7844 -184
-7062 -2640
-8577 -3443
-4833 -7210
729 -4914
7483 -2872
-9912 -2567
-7989 -8836
-7681 -6446
-5889 -9822
-9461 -587
-4221 -3593
8310 8932
-7363 3578
2776 4264
-8357 379
-8498 9860
-3596 -5893
-7689 7157
4778 7526
-8827 -8146
-4052 2887
-9762 -2564
-957 -932
-2462 1544
-6317 -9644
-2216 717
1255 -9159
7572 5745
-9184 4876
6713 -9046
2043 -1250
-6434 -9797
-2467 7943
2954 -364
4199 -767
-6996 6312
7235 1385
8078 -1707
-3417 6248
5956 -6035
-3944 -3872
-2669 6251
-4058 8880
-3803 -1557
-3295 1899
8157 -6248
-1910 -3717
-1391 -6667
-6945 4176
1957 -6782
3512 -3985
-7465 -1417
-2172 -6461
-3088 -8519
-4587 -1770
4965 -2552
2605 767
6205 104
9472 8183
-4810 1318
9950 9251
-9077 -6484
8197 -6361
-4551 673
9391 -8630
-1479 7499
-2837 6545
9705 3991
-7999 -6359
6899 5791
7299 9530
2930 3031
-9202 9263
-9934 -1837
-3704 -3397
7941 -6797
-9008 -618
-3749 7205
-8785 6646
-2285 -9802
-2503 5954
-388 -636
7055 1116
6668 5549
-2232 -746
5838 9528
-8574 6048
695 -673
5238 210
4267 -3448
-9919 -679
4096 4207
-9594 9792
-5679 -9459
-8122 1404
-2617 6668
-2136 2450
2963 4426
-1493 6619
6506 -3950
1541 -231
-2212 -3752
8927 1764
2746 5420
5598 6574
7898 472
-7551 4967
-8695 -2664
-3720 -6929
5914 3577
-7958 -3479
9027 8203
-4452 -7692
-6054 -2878
-3363 6376
1254 444
-9125 5310
-8116 -2543
839 -6768
6167 5206
-1546 -7890
7333 5279
6606 8586
700 5078
-3230 7081
-3202 4432
7534 2904
7832 -2094
8893 8199
-8455 -7701
-1591 -7424
4859 -8909
-5244 1626
7655 4382
-6471 7680
3903 7413
5038 -3507
-1500 -6181
3068 4501
8246 -9671
1980 -1008
8910 1965
-872 936
-2938 -2313
-6805 -1815
8040 9294
-8478 -9607
-429 7648
8796 4932
-8682 -6345
8214 -6531
-9371 8174
9224 -9925
1007 9173
8100 -2387
-6520 9038
8283 -8919
-386 8414
-9591 736
-755 5069
7800 7151
4492 -3334
-2334 -526
-371 5474
-5482 -3004
9150 -3062
-8213 -3735
-6554 9656
-9746 -4678
996 -2064
-8423 -5955
-1591 -156
-5062 4624
940 8851
-2897 9169
-7699 -3969
-5049 9258
-7743 958
5876 -6416
-9095 6292
-531 -7390
3454 6722
-3904 -7834
4687 5437
407 9904
4022 -8472
-8606 -1006
-9045 9647
6166 2971
-5893 -6568
-8007 -5340
-4901 7266
-8979 9456
-6760 3417
-4705 4931
-5401 6628
5500 -3124
1478 -3750
-1224 -2561
943 -6391
9842 7093
1142 -6821
2213 2136
-6944 5988
-6782 -2284
3283 6775
-997 -8203
8653 2637
7026 3795
-8557 2001
35 3230
1395 6813
6425 8401
-9241 -8324
-899 2851
5318 -7087
4412 -6909
7339 -1231
-3988 2332
6798 -9070
9386 -2628
-2956 7656
5428 -7197
3138 2681
-3872 7422
2174 584
606 -1556
-7889 -9815
4708 -3692
922 8761
9481 1489
-1344 -3032
8573 2788
-4798 4953
6622 -8726
3367 -4715
4721 4962
-3168 -3833
-7886 3547
-7090 6686
9306 -6060
-118 4365
6355 213
8436 2449
-357 969
6661 -3238
-2532 -4170
-3102 5509
-9131 6068
1472 -1638
9033 4873
969 -5375
-697 2611
7311 -628
-5588 -2039
724 -7263
-9946 -1565
9159 -6813
3463 5997
3714 4363
-6271 -9939
8853 -1402
-655 -5206
-8078 -9905
-9199 -6022
-3910 -1682
89 9252
-5198 3607
135 -3295
-4355 368
3520 -8562
3932 8566
1318 -8280
9314 -7750
5786 2995
-7564 0
4709 4746
7399 -705
9481 -6575
-8968 -6869
-8587 -2275
8417 -204
-4358 -2761
1884 5994
8233 4841
913 -888
-5630 -2327
-4362 8145
2291 9096
9608 -2677
6718 4368
-5731 -6908
-2256 4702
6085 1640
-4660 -8189
-7418 -4260
-8565 1490
378 -9926
-7196 6227
-8733 -24
-5480 7572
-6278 -6103
-7733 -7964
5738 2889
941 9023
-8591 397
7193 2673
-1373 -9408
-509 6914
3431 7166
9220 8795
-9913 -6018
-245 -7334
-1603 4172
-8695 -5119
8999 -7469
-9868 7609
-2087 -1542
1982 -4836
-4998 -5876
-6306 6585
4447 8988
710 -6286
-7566 -9018
4601 -4637
4605 -7317
-1146 650
1297 -1256
-8354 -4449
-6127 -4348
-9492 4117
-8552 -8461
-8768 -8210
7151 288
9896 5933
456 -5144
-1212 -1990
3915 -3218
-5747 -4154
-3589 -7785
-4060 5443
-1622 7924
-4353 -9298
-4791 9222
3526 3669
992 2344
-1030 2424
-6216 -6851
1849 5811
1836 3253
8745 -64
6039 5863
-6549 5457
6108 7499
8563 3721
-3962 851
3073 -7591
-2654 474
7273 2145
-4907 -2674
3092 1118
5491 -3902
2145 -3116
-9559 3229
-6686 -5667
-2612 875
9659 2793
-6227 1627
-1798 5572
3834 -9155
-1558 1626
-5471 6481
