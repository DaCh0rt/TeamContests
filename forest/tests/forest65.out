81298.26 436313569.32
